guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+* RPM repositories
+* Kickstart repositories
+* File repositories
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the rpms, listing files and metadata of each repository in yum format that could then be used synchronize in the importing {ProjectServer}.
```

Comment:
```suggestion
The export of RPM repositories contains directories containing the packages, listing files, and metadata of each repository in Yum format.
You can use this information to import the content view version into {ProjectServer}.
```

Does this also apply to Kickstart and File repositories?

Reply:
And I'd format `listing` as code to make it look like a literal.

Reply:
> Does this also apply to Kickstart and File repositories?

Listing files are generated for yum/kickstart only. In fact they are generated just for RH content not custom.